dotCMS/src/main/java/com/dotmarketing/util/Config.java

```diff
@@ -298,137 +297,78 @@ private static void _refreshProperties () {
+		if (propsArr == null || propsArr.length == 0) {
+		    return defValue;
+		} 
+		
+		return String.join(",", propsArr);
+		
 	}
 
 	/**
-	 * Returns a string property. The {@link Supplier} is useful when you want
-	 * lazy evaluation for the default value. This means that figuring out the
-	 * value of the supplier will not happen until the logic determines that the
-	 * property specified by the {@code name} parameter does not map to a value
-	 * in the properties files.
-	 *
+	 * this is only here so the test passes
 	 * @param name
-	 *            - The name of the property to read.
 	 * @param defValue
-	 *            - The default value as a {@link Supplier}, in case the
-	 *            property is not defined.
-	 * @param forceDefaultToString
-	 *            - If the default value is to be returned when the property is
-	 *            not defined in the configuration files, set to {@code true}.
-	 *            Otherwise, set to {@code false}.
-	 * @return The value of the specified property, or its default value.
-	 */
-	public static String getAsString(String name, Supplier<String> defValue, boolean forceDefaultToString) {
-		_refreshProperties();
-		String result = StringUtils.EMPTY;
-		if (props != null) {
-			String[] propsArr = props.getStringArray(name);
-			StringBuilder property = new StringBuilder();
-			if (propsArr != null && propsArr.length > 0) {
-				buildProperty(propsArr, property);
-				result = property.toString();
-			} else if (forceDefaultToString) {
-				result = (defValue != null) ? defValue.get() : "";
-			}
-		} else {
-			// Default is not forced to string here for historical reasons.
-			// Presumably props are never actually null.
-			result = (defValue != null) ? defValue.get() : "";
-		}
-		return result;
-	}
-
-	/**
-	 * 
-	 * @param propsArr
-	 * @param property
+	 * @param thing
+	 * @return
 	 */
-	private static void buildProperty(String[] propsArr, StringBuilder property) {
+	@Deprecated
+    public static String getStringProperty(final String name, final String defValue, boolean thing) {
 
-		int i = 0;
+        return getStringProperty(name, defValue);
+        
+    }
 
-		for (String propItem : propsArr) {
-            if(i > 0) {
-                property.append(",");
-            }
-            property.append(propItem);
-            i++;
-        }
-	}
+	
+	
+	
 
 	/**
 	 * @deprecated  Use getStringProperty(String name, String default) and
 	 * set an intelligent default
 	 */
 	@Deprecated
     public static String getStringProperty (String name) {
-        _refreshProperties ();
-        String[] propsArr = props.getStringArray(name);
-        String property = new String ();
-        int i = 0;
-        for (String propItem : propsArr) {
-            if (i > 0)
-                property += ",";
-            property += propItem;
-            i++;
-        }
-        return property;
+        String value = getStringProperty(name, null);
+
+        return value;
```

Comment:
Codacy found an issue: [Consider simply returning the value vs storing it in local variable 'value'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5943250)